test/e2e/bots_test.go

```diff
@@ -353,6 +423,9 @@ func runBotTest(t *testing.T,
 				Use "helm [command] --help" for more information about the command.`))
 
 			expectedMessage := fmt.Sprintf("%s\n%s", cmdHeader(command), expectedBody)
+			if botDriver.Type() == commplatform.SlackBot {
+				expectedMessage = fmt.Sprintf("%s %s", cmdHeader(command), expectedBody)
+			}
```

Comment:
what if we will just hide it in the `botDriver.WaitForMessagePosted`, so in slack it will be at the beginning:
```go
expectedMsg = strings.Replace(expectedMsg, "\n", " ", 1)
```

Reply:
We have plenty of test cases in other tests, so doing this trick might be solve only this case but break others. Maybe more long term solution would be advantageous as you said.